docs/code-organization/git-quickstart.mdx

```diff
@@ -0,0 +1,331 @@
+---
+sidebar_position: 1
+---
+
+# Git Quickstart
+
+This page contains [Git][git-link] commands that may be useful
+during development. Examples have been provided where the command to be
+executed is prefixed with `$` and the sample output are not.
+
+Please refer to the [documentation for Git][git-doc] for a
+deeper understanding in Git.
```

Comment:
let's add link to the git setup doc at the top: https://docs.ntuscse.com/docs/getting-started/github